bigtable/integration_test.go

```diff
@@ -2105,6 +2131,36 @@ func TestIntegration_AdminBackup(t *testing.T) {
 	if _, err := adminClient.TableInfo(ctx, restoredTable); err != nil {
 		t.Fatalf("Restored TableInfo: %v", err)
 	}
+	// Set backup policy.
+	// Get Etag first from GetIAMPolicy.
```

Comment:
From [documentation](https://cloud.google.com/bigtable/docs/reference/admin/rest/v2/Policy): 
> It is strongly suggested that systems make use of the etag in the read-modify-write cycle to perform policy updates in order to avoid race conditions: An etag is returned in the response to getIamPolicy, and systems are expected to put that etag in the request to setIamPolicy to ensure that their change will be applied to the same version of the policy.